### PR TITLE
docs: Update list of project types in comments for config.yaml, fixes #5544

### LINKED ISSUE #5544: New project template: Projecttypes are not uptodate
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Output of `ddev debug test`

_No response_

### Expected Behavior

Is it probably too difficult to read out the project types dynamically? 
I would simply extend the list and also refer to https://ddev.readthedocs.io/en/latest/users/quickstart/.

### Actual Behavior

Just created a new project with 1.22.2. I noticed that the project types in the template are not up to date. See also list of hardcoded projecttypes: https://github.com/ddev/ddev/blob/master/pkg/ddevapp/templates.go#L10

### Steps To Reproduce

Just create a new project with current ddev version

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ayacoo**
* #5544

### CODE REVIEW

File: pkg/ddevapp/templates.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ const ConfigInstructions = `
 #   http://projectname.ddev.site and https://projectname.ddev.site
 
 # type: <projecttype>  # drupal6/7/8, backdrop, typo3, wordpress, php, craftcms, laravel, magento2, shopware6, silverstripe
```
Comment (rfay) [inline pkg/ddevapp/templates.go:10]:
```suggestion
# type: <projecttype>  # drupal6/7/8/9/10, backdrop, typo3, wordpress, php, craftcms, laravel, magento2, shopware6, silverstripe
```

**Review by rfay — CHANGES_REQUESTED** [review]
Just missed a thing or two. Did you want magento in there also? 

The official list is `backdrop, craftcms, django4, drupal10, drupal6, drupal7, drupal8, drupal9, laravel, magento, magento2, php, python, shopware6, silverstripe, typo3, wordpress`

File: pkg/ddevapp/templates.go [hunk 2]
```diff
@@ -7,7 +7,9 @@ const ConfigInstructions = `
 # name: <projectname> # Name of the project, automatically provides
 #   http://projectname.ddev.site and https://projectname.ddev.site
 
-# type: <projecttype>  # drupal6/7/8, backdrop, typo3, wordpress, php
+# type: <projecttype>  # drupal6/7/8, backdrop, typo3, wordpress, php, craftcms, laravel, magento2, shopware6, silverstripe
+# See https://ddev.readthedocs.io/en/latest/users/quickstart/ for more
+# information on the different projecttypes
```
Comment (rfay) [inline pkg/ddevapp/templates.go:12]:
```suggestion
# information on the different project types
```

**Comment by ayacoo** [comment]
Firstly, thank you for your support :-)

I was confused by the documentation here because project type "magento" was not mentioned there: https://ddev.readthedocs.io/en/latest/users/quickstart/#magento (tab "OpenMage/Magento 1")

Django is tagged as "experimental". That's why I was unsure.

I have taken the liberty of adopting your list, including alphabetical sorting. This is otherwise rather difficult to read

**Review by rfay — APPROVED** [review]
Thanks!

**Pull request merged**